africanmathsinitiative/R-Instat
# Improved usability of the Survival Object dialog

Fixes #6165

Changes made:
* Added "Type" label to the top of the dialog for the buttons
* Changed an input into a combo box when a numeric variable is selected
* Added "Type of Censoring" into the summary output
* Added "Censored when status is ..." into the summary output

To test:
* Use a data set from the "Survival" package in the R library (e.g. `cancer`, `cbd`)
* Access the dialog by `Structured > Survival > Define`
* It should create a survival object which is not visible (except through the `View Objects` dialog. The output summarises the object created

@africanmathsinitiative/developers this is ready for review
